packages/odyssey-react-mui/src/RadioGroup.tsx

```diff
@@ -73,31 +89,79 @@ const RadioGroup = ({
   HintLinkComponent,
   id: idOverride,
   isDisabled,
+  isReadOnly = false,
   label,
   name: nameOverride,
   onChange: onChangeProp,
   testId,
   translate,
   value,
 }: RadioGroupProps) => {
+  const [internalValue, setInternalValue] = useState(value ?? defaultValue);
```

Comment:
Per Slack convo: With these other change being made, we no longer need any internal value as far as I can tell